src/LED.h

```diff
@@ -53,6 +55,8 @@ class LED
 protected:
   byte _pin;
   byte _mode;
+  bool _active;
+  bool _state;
```

Comment:
Do you really need _state as a member variable? Seems to do the same thing as _mode above. 
_state is only used in _update(). Could it be local variable there?

Reply:
Yes, it makes more sense to make it a local variable.